packages/text/src/html.ts

```diff
@@ -33,3 +34,48 @@ export function parseHTML (content: string, extensions: Extensions): ProseMirror
 
   return ProseMirrorNode.fromJSON(schema, json)
 }
+
+const ELLIPSIS_CHAR = '…'
+const WHITESPACE = ' '
+
+/**
+ * @public
+ */
+export function stripTags (htmlString: string, textLimit = 0, extensions: Extensions | undefined = undefined): string {
+  const effectiveExtensions = extensions ?? defaultExtensions
+  const parsed = parseHTML(htmlString, effectiveExtensions)
+
+  const textParts: string[] = []
```

Comment:
Why not use `parsed.textContent`?

Reply:
Imagine you have block elements at start of your comment. Like <li> or <h2>. parsed.textContent gives you plain text without spaces between such blocks. Text from different block elements will be concatenated and glued together. 

For input 
`<h1 class="proseHeading">Header</h1><p>Paragraph 1 text</p><ol><li><p>Item 1</p></li><li><p>Item 2</p></li></ol>`

You will get the following `parsed.textContent`
` HeaderParagraph 1 textItem 1Item 2`

Or simpler, if you will have very short first paragraph.

I handled such cases.